README.md

```diff
@@ -2,42 +2,1104 @@
+            "id": {
+              "type": "string",
+              "description": "The customer identifier.",
+              "name": "ID",
+              "kind": "ATTRIBUTE",
+              "required": true,
+              "displayName": "Identifier",
+              "summary": "Inline description",
+              "comments": "come sopra",
+              "examples": [
+                "1234567",
+                "988654"
+              ],
+              "status": "come sopra",
+              "tags": [
+                "tag1",
+                "tag2"
+              ],
+              "externalDocs": "https://www.google.it/",
+              "default": null,
+              "isClassified": true,
+              "classificationLevel": "INTERNAL",
+              "isUnique": true,
+              "isNullable": false,
+              "pattern": "^[0-9]+$",
+              "format": "named pattern e.g. email",
+              "enum": [
+                "VAL1",
+                "VAL2"
+              ],
+              "minLength": 2,
+              "maxLength": 10,
+              "contentEncoding": "UTF-8",
+              "contentMediaType": "application/json",
+              "precision": 0,
+              "scale": 10,
+              "minimum": 0,
+              "exclusiveMinimum": true,
+              "maximum": 10000,
+              "exclusiveMaximum": false,
+              "readOnly": true,
+              "writeOnly": true,
+              "physicalType": "VARCHAR",
+              "partitionStatus": true,
+              "partitionKeyPosition": 2,
+              "clusterStatus": true,
+              "clusterKeyPosition": 2,
+              "ordinalPosition": 0
+            }
+          }
+        }
+      ]
+    }
+  },
+  "components": {
+    "serverInfo": {
+      "flightFrequencyStoreServerInfo": {
+        "host:": "{host}",
+        "port:": "3306",
+        "dbmsType:": "MySQL",
+        "dbmsVersion:": "8",
+        "connectionProtocols": {
+          "jdbc": {
+            "version": "1.0",
+            "url": "jdbc:mysql://{hosts}:3306/foodmart",
+            "driverName": "MySQL JDBC Driver",
+            "driverClass": "org.mysql.Driver",
+            "driverVersion": "latest",
+            "driverLibrary": {
+              "description": "MySQL JDBC Driver Library",
+              "dataType": "application/java-archive",
+              "$href": "https://jdbc.mysql.org/"
+            },
+            "driverDocs": {
+              "description": "MySQL JDBC Driver HomePage",
+              "dataType": "text/html",
+              "$href": "https://jdbc.mysql.org/mysql-8.jar"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### Async Api
+
+#### Raw Port Async Api V3
+This section illustrates an example of an Async API version 3 configuration for a Trip Status Streaming API. The API is used to stream events related to the Trip entity. It describes the schema for the messages, including the fields and their types.
```

Comment:
Non commenti quali risorse vengono create in Blindata a fronte di questo descriptor.